src/docker.test.ts

```diff
@@ -47,14 +47,18 @@ describe("Docker images", (): void => {
     cache.restoreCache.mockResolvedValueOnce(cacheHit ? KEY : undefined);
     await docker.loadDockerImages();
 
-    expect(core.getInput).lastCalledWith("key", { required: true });
+    expect(core.getInput).lastCalledWith("key", { required: true }); // TODO: or read-only
     expect(cache.restoreCache).lastCalledWith([util.DOCKER_IMAGES_PATH], KEY);
   };
 
-  const mockedSaveDockerImages = async (cacheHit: boolean): Promise<void> => {
+  //helper function to save docker images
```

Comment:
This is a fairly redundant comment. Suggest either expanding or simply dropping it. Also, remember the space and capitalization after `//`.